static/app/components/events/eventMessage.tsx

```diff
@@ -47,14 +48,17 @@ function EventMessage({
   message,
   type,
   showUnhandled = false,
+  hideLevel = false,
 }: Props) {
   return (
     <LevelMessageContainer className={className}>
-      <EventOrGroupLevel
-        level={level}
-        levelIndicatorSize={levelIndicatorSize}
-        type={type}
-      />
+      {!hideLevel && (
```

Comment:
good catch  - will not pass in level for now, will deal with an updated design for level in a different PR (since it also likely will affect the issue stream)